### DISCUSSION TITLE
How to deal with space-separated color values without postcss-preset-env

### DISCUSSION THREAD

**Discussion by dagoss** [parcel-bundler/parcel#8092] [Q&A]
I'm having an issue where I need to support a browser that doesn't support space-separated color values (e.g. `rgb(255 255 255 / 0.5)`). I have the browser, Edge 18, listed in my browserslist:

    "browserslist": "> 0.5%, last 2 versions, not dead, Firefox ESR, Edge 18"

I'm still getting space-separated colors in the output. I installed the postcss-preset-env plugin to take care of it (and that solved my issue), but now I see a warning about having a redundant plugin. According to the docs, Parcel has the equivalent of postcss-preset-env so I should need that plugin.

Is there something else I need to do to get this to work without postcss-preset-env?

**Comment by mischnic** [comment]
Which Parcel version are you using?

**Reply by dagoss** [reply]
2.5.0

**Reply by mischnic** [reply]
I've just tested this, and for your browserslist and
```css
body {
	color: rgb(255 255 255 / 0.5);
}
```
, I get this output: `body{color:rgba(255,255,255,.5)}` with `parcel build index.css`

**Reply by dagoss** [reply]
Interesting. The space-separated colors I'm seeing in my output are from [Tailwind](https://tailwindcss.com/) CSS classes. I followed Tailwind's [installation for Parcel instructions](https://tailwindcss.com/docs/guides/parcel).

If I put space-separated colors in my project's CSS, you are correct, Parcel is appropriately taking care of them base on the browserslist, but the ones that are being pulled in from Tailwind are not unless I include the postcss-preset-env plugin.

**Reply by mischnic** [reply]
> but the ones that are being pulled in from Tailwind

Which tailwind classes cause this? I should be able to reproduce it then

**Reply by dagoss** [reply]
> > but the ones that are being pulled in from Tailwind
> 
> Which tailwind classes cause this? I should be able to reproduce it then

Pretty much any class with with a color value (`.text-white` for example).

**Reply by mischnic** [reply]
Appears to be a bug related to colors containing css variables: https://github.com/parcel-bundler/parcel-css/issues/178
So it doesn't matter if it's coming from tailwind or now.
